src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -473,4 +475,43 @@ public void testGenotypingForSomaticGVCFs_subsetAlts() {
             Assert.assertTrue(vc.getNAlleles() <= 3);  //NAlleles includes ref
         }
     }
+
+    @Test
+    public void testRawAndFinalizedAlleleSpecificAnnotationsThoroughly() {
```

Comment:
Have you confirmed that this test fails without your bug fixes?

Reply:
It sure does.